pkg/oci/utils.go

```diff
@@ -39,6 +39,16 @@ var (
 	BundlePathKey = "com.github.containers.virtcontainers.pkg.oci.bundle_path"
 )
 
+type CompatOCIProcess struct {
```

Comment:
Right, after some digging I found it.

@sboeuf - it would have been helpful to reference this: https://github.com/opencontainers/runtime-spec/pull/675 and/or https://github.com/opencontainers/runtime-spec/commit/37391fb66ffc7f2bc87c1b85349b68b561c53627 explicitly.